src/helpers/buffers/fsv.rs

```diff
@@ -29,7 +31,7 @@ impl<const N: usize> FixedSizeByteVec<N> {
             data: vec![0_u8; N * capacity.get()],
             added: bitvec![0; capacity.get()],
             capacity,
-            end: capacity.get(),
+            end: AtomicUsize::new(capacity.get()),
```

Comment:
It looks like `ordering_mpsc` is a replacement for `fsv`, so I am wondering why we need this change?

Reply:
Ah, because I started by modifying fsv, and forgot to revert this, I missed than in my self-review.  Will revert.